pulseguardian/guardian.py

```diff
@@ -5,19 +5,28 @@
 import logging
 import re
 import time
+import requests
 
 from pulseguardian import config
 from pulseguardian.logs import setup_logging
 from pulseguardian.management import PulseManagementAPI
 from pulseguardian.model.base import init_db, db_session
-from pulseguardian.model.user import PulseUser
+from pulseguardian.model.user import PulseUser, User
 from pulseguardian.model.queue import Queue
-from pulseguardian.sendemail import sendemail
+from pulseguardian.sendemail import sendemail as email
 
 logging.getLogger("requests").setLevel(logging.WARNING)
 
 setup_logging(config.guardian_log_path)
 
+def sendemail(to_addrs=[], subject=None, text_data=None, from_addr=config.email_from,
+                username=config.email_account, password=config.email_password,
+                server=config.email_smtp_server, port=config.email_smtp_port,
+                use_ssl=config.email_ssl):
+    email(subject=subject, from_addr=from_addr, to_addrs=to_addrs, username=username,
```

Comment:
A nice cleanup here would be to actually only take `to_addrs`, `subject`, and `text_data` here, because we actually never override the other parameters. You can always send those default parameters to the `sendemail.sendemail()` function.